src/D3MHub.sol

```diff
@@ -331,6 +333,16 @@ contract D3MHub {
         emit Unwind(ilk, amount, fees);
     }
 
+    function _fix(bytes32 ilk, address _pool) internal returns (uint256 diff) {
+        (uint256 ink, uint256 art) = vat.urns(ilk, _pool);
+        if (art < ink) {
+            address _vow = vow;
+            diff = _min(ink - art, MAXINT256);
```

Comment:
If the diff is somehow greater than `MAXINT256` we should probably revert, as even if we fix `MAXINT256` we keep our invariant of `ink==art` broken.
I think such a big diff can only be in some attack scenario, in which case we probably want to block `exec` anyway and have governance try to handle the case.

Reply:
Actually I think we should remove the `_min` at all. In `_wind` there is a protection to not leave the position > MAXINT256, so if there isn't a contract bug is impossible to get to that state. We should also be able to remove it from `_unwind`.

Reply:
yeah, it can be removed.

Reply:
I had made a long comment about it and not sure what happened, if was removed somehow, or not even sent.
Anyway what I was saying is that I'm actually not sure what to do it here. In theory as we are blocking in `wind` to be `> MAXINT256`, then we should be able to remove any other check. However that is relying 100% than an external source won't alter it, the invariant can be broken as the storage doesn't happen here but in the `vat`.
Thinking on that is nice to have checks that ensure no overflow can happen at all, just as a best practice. That can be done with `require`s as well but we had decided to remove it from `unwind` to avoid any case of stopping it.
Putting it here, kind of breaks that reasoning, so I think we have these different scenarios:
1) Remove all the checks at all and rely an external source won't make this invariant to break
2) Add just requires for best practice as they won't trigger anyway if there isn't an external intervention.
3) Leave the min and in case it would trigger (which shouldn't happen without external intervention) the imbalance will be fixed partially.

I think I prefer option 2.

Reply:
agreeing on 2, that was my original preference.

Reply:
Done: https://github.com/makerdao/dss-direct-deposit/pull/70/commits/677306e6971858795f7a21cfb3fb41505a95f732

Reply:
looks good, resolving this.